styles.css

```diff
@@ -1,29 +1,311 @@
+:root {
+  --text-brown: #f4a460;
+  --bg-brown: rgb(155, 104, 68);
+  --hover-brown: #e5d3b3;
+}
+
+html,
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  font-family: monospace;
+}
+
 *,
 :before,
 :after {
   box-sizing: border-box;
-  border: red 1px solid;
+  /* border: red 1px solid; */
 }
+
 body {
   background-color: #9b6844;
+  height: 100vh;
+  max-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
 }
 h1,
+h2,
 h3,
-a {
+a,
+label,
+#submitbtn {
   text-align: center;
   font-family: monospace;
-  color: SandyBrown;
+  color: var(--text-brown);
 }
+
 .container {
-  margin: 80px auto;
-  width: 350px;
+  margin: 0 auto;
+  max-width: 350px;
+  width: 100%;
 }
 .baby-groot {
   position: relative;
   display: flex;
   justify-content: center;
   flex-direction: column;
+  margin: 0 auto;
+  max-width: 80vw;
+}
+
+footer {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  height: auto;
+}
+
+.me-tsvstacks p {
+  margin: 0;
+}
+
+.me-tsvstacks {
+  border-radius: 5px;
+  padding: 5px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.fa-icons a {
+  text-decoration: none;
+}
+
+.me-tsvstacks,
+.me-tsvstacks a {
+  color: black;
+  /* text-shadow: -1px -1px 0 #000, 1px -1px 0 #000, -1px 1px 0 #000, */
+  /* 1px 1px 0 #000; */
+  font-size: 16px;
+  text-decoration: none;
+}
+
+.me-tsvstacks a {
+  color: black;
+}
+
+.modal-container {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  padding: 0 15%;
+}
+
+.fade-out {
+  animation: fadeOut ease 2s;
+  animation-fill-mode: forwards;
+}
+
+.fade-in {
+  animation: fadeIn ease 2s;
+  animation-fill-mode: forwards;
+}
+
+.sad {
+  margin-top: 25px;
+  transform: rotate(180deg) !important;
+}
+
+#winner {
+  display: flex;
+  display: none;
+}
```

Comment:
Not sure whats going on here... you have two declarations for the `#winner` id. Best way to fix this would be to look at what classes are actually being used and are desired in a browser and removing all those that are not doing anything, then consolidate all the properties into one declaration. If there are any other classes like this the same should apply